plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-actions.php

```diff
@@ -88,6 +88,31 @@ public static function output( $post ) {
 		<?php
 	}
 
+	/**
+	 * Forms a trash/delete order URL.
+	 *
+	 * @param int $order_id The order ID for which we want a trash/delete URL.
+	 *
+	 * @return string
+	 */
+	private static function get_trash_or_delete_order_link( int $order_id ): string {
+		if ( OrderUtil::custom_orders_table_usage_is_enabled() ) {
+			$order_list_url  = admin_url( 'admin.php?page=wc-orders' );
+			$trash_order_url = add_query_arg(
+				array(
+					'action'           => 'trash',
+					'order'            => array( $order_id ),
```

Comment:
Not that I'm reviewing this PR but was taking a quick look in relation with #34229 and noticed we're using `id` instead of `order` for the edit page (e.g. `admin.php?page=wc-orders&id=5894&action=edit`). Shouldn't we do the same here?

Reply:
It works because the checkbox column (in the list table) uses `order[]` and that is also what the bulk action handler looks for. Perhaps it would be nice to change things so that we have greater consistency† ... whether we should consolidate on `id` or `order`, though, I'm less sure. Here's a take on that, though:

- The work to [introduce the new list table](https://github.com/woocommerce/woocommerce/pull/32864) was completed before the new editor screen.
- When that initial work was done, we picked `order` and `orders` as the singular and plural object names.
- This felt like a good analog for the CPT list table, where `post` is used (instead of `ID`).

† *Consistency is good, generally, but I'd also note in the case of taxonomy list tables WordPress uses `tag_ID` for the edit links and `delete_tags[]` for bulk actions, so what we currently have isn't so unusual.*

What are your thoughts? I don't personally lean very strongly in either direction.

Reply:
Yeah. I get where this is coming from, just thought it was "funny" that we're using different names (CPT at least "consistently" uses `post`).

> What are your thoughts? I don't personally lean very strongly in either direction.

I don't either, though I tend to prefer "order". Maybe this is something we can decide later or add some support for re-mapping some of the args in a given context (for example, translate `?action=trash&id=1` to `?action=trash&order[]=1`).

In the mean time, I've used the same approach as here for #34229.

Reply:
... which, as luck would have it, was assigned to you 😅.

Reply:
🙇🏽 

Cool; we definitely have some time before things are really 'locked down' to revise areas like this.

Reply:
<facepalm> On further reflection, I guess `order` as a query arg is not ideal, as it has a lot of potential for confusion with the query arg used for ordering direction (asc, desc).

I'll create a PR to address this, probably after #34229 is merged to avoid unnecessary work there.